kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java

```diff
@@ -360,6 +380,19 @@ public String marshall(Object objectInput) {
         }
     }
 
+    @Override
+    public <T> T unmarshall(byte[] serializedInput, Class<T> type) {
```

Comment:
why is this required ?

Reply:
this is used in RawJsonReader It is not strictly needed, but save a byte[]-string conversion, which provide some peformance gain

Reply:
move to the marshaller interface with default implementation UnsupportedOperationException

Reply:
it is int he marshaller with default implementation to call the string one